services/github/github-all-contributors.service.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { renderContributorBadge } = require('../contributor-count')
+const { ConditionalGithubAuthV3Service } = require('./github-auth-service')
+const { fetchJsonFromRepo } = require('./github-common-fetch')
+const { documentation } = require('./github-helpers')
+
+const schema = Joi.object({
+  contributors: Joi.array(),
```

Comment:
We need to ensure that the contributors array always exists as part of the response validation, so let's add a `.required()`
```suggestion
  contributors: Joi.array().required(),
```